### PR TITLE
feat(MOPLAT-353): Show availability of artworks

### PR DESCRIPTION
**Pull request by patrinoua**
### Description

This PR resolves [MOPLAT-353]

https://user-images.githubusercontent.com/36475005/185917363-24da6347-1021-4b77-85fa-59efd69def07.mov

### PR Checklist
- [ ] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
### To the reviewers 👀
- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-353]: https://artsyproduct.atlassian.net/browse/MOPLAT-353?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/sharedUI/items/ArtworkGridItem.tsx [hunk 1]
```diff
@@ -26,7 +27,7 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = (props) => {
           }}
         />
         <Text italic variant="xs" color="black60" mt={1}>
-          {artwork.title},{" "}
+          <AvailabilityDot availability={artwork.availability || ""} /> {artwork.title},{" "}
```
Comment (rajsam003) [inline src/app/sharedUI/items/ArtworkGridItem.tsx:30]:
```suggestion
          <AvailabilityDot availability={artwork.availability} /> {artwork.title},{" "}
```
Lets avoid using "" and fix the type in <AvailabiltyDot/> file. Also using a switch case will help us.

File: src/app/sharedUI/atoms/StatusDot.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  availability?: string
+}
+
+export const AvailabilityDot = ({ availability }: AvailabilityDotProps) => {
+  let color = "black60"
+  if (availability == "for sale") color = "green100"
+  if (availability == "on hold") color = "orange100"
+  if (availability == "sold") color = "red100"
+
+  return <Flex height={10} width={10} backgroundColor={color} borderRadius="10" />
```
Comment (rajsam003) [inline src/app/sharedUI/atoms/StatusDot.tsx:13]: **non-blocking** : We can use `switch` instead of `if-else` because its faster.

**Comment by rajsam003** [comment]
Good work. 
I just added a few non-blocking comments/suggestions.

File: src/app/sharedUI/atoms/StatusDot.tsx (hunk 2 above)
Comment (patrinoua) [reply src/app/sharedUI/atoms/StatusDot.tsx]: Good point, thank you! Just updated it.

**Comment by patrinoua** [comment]
@rajsam003 thank you for reviewing, I've added your suggestions, feel free to merge if it's ok :)

**Review by rajsam003 — APPROVED** [review]
Awesome 👏

**Pull request merged**